datus/agent/node/gen_metrics_agentic_node.py

```
@@ -0,0 +1,696 @@
+            unique_trees = set()
+            for item in results:
+                d = item.get("domain")
+                l1 = item.get("layer1")
+                l2 = item.get("layer2")
+
+                if d and l1 and l2:
+                    unique_trees.add(f"{d}/{l1}/{l2}")
+
+            return sorted(list(unique_trees))
+
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)